baseline_stability.py

```diff
@@ -1,68 +1,32 @@
-import numpy as np
-from digicampipe.io.event_stream import event_stream
-import digicampipe.io.zfits as zfits
-from digicampipe.utils import geometry
-from cts_core.camera import Camera
-from digicamviewer.viewer import EventViewer
-from os.path import expanduser
-from optparse import OptionParser
-import matplotlib.pyplot as plt
-import os.path
-
-
-if __name__ == '__main__':
-
-    home_absolute_path = expanduser('~')
-    parser = OptionParser()
-    parser.add_option("-p", "--path", dest="directory", help="directory to data files",
-                      default=os.path.realpath('digicampipe/tests/resources'))
-    parser.add_option('-f' , '--file', dest='filename', help='file basename e.g. CRAB_01_0_000.%03d.fits.fz',
-                      default='/example_100_evts.%03d.fits.fz', type=str)
-    # parser.add_option("-o", "--output", dest="output", help="output filename", default="output_crab.txt", type=str)
-    # parser.add_option("-d", "--display", dest="display", action="store_true", help="Display rather than output data",
-    #                   default=False)
-    parser.add_option('-s', "--file_start", dest='file_start', help='file number start', default=0, type=int)
-    parser.add_option('-e', "--file_end", dest='file_end', help='file number end', default=0, type=int)
-    parser.add_option('-c', "--camera_config", dest='camera_config_file', help='camera config file to load Camera()'
-                      , default='digicampipe/tests/resources/camera_config.cfg')
+'''
+Something with baseline stability
 
-    (options, args) = parser.parse_args()
+Usage:
+  baseline_stability <fitsfile>
 
-    # Input/Output files
-    directory = options.directory
-    filename = directory + options.filename
-    file_list = [filename % number for number in range(options.file_start, options.file_end + 1)]
-    digicam_config_file = options.camera_config_file
-
-    # Camera and Geometry objects (mapping, pixel, patch + x,y coordinates pixels)
-    digicam = Camera(_config_file=digicam_config_file)
-    digicam_geometry = geometry.generate_geometry_from_camera(camera=digicam)
-
-    # Noisy pixels not taken into account
-    # pixel_not_wanted = [1038, 1039, 1002, 1003, 1004, 966, 967, 968, 930, 931, 932, 896]
-    n_events = zfits.count_number_events(file_list=file_list)
-    ####################
-    ##### ANALYSIS #####
-    ####################
-    digicam_baseline = np.zeros((len(digicam_geometry.pix_id), n_events))
+Options:
+  -h --help     Show this screen.
+'''
+from digicampipe.io.event_stream import event_stream
+import matplotlib.pyplot as plt
+from docopt import docopt
+from tqdm import tqdm
 
-    # Define the event stream
-    data_stream = event_stream(file_list=file_list, expert_mode=True, camera_geometry=digicam_geometry, camera=digicam)
 
-    for i, data in zip(range(n_events), data_stream):
+def main(infile):
 
+    baselines = []
```

Comment:
I would prefer to initiate it as a numpy array as it was before: 
` digicam_baseline = np.zeros((len(digicam_geometry.pix_id), n_events)) `

then we can work on mean, std max and so on...
P.S. don't forget to change the append() in the for loop.

Reply:
we can merge if you don't what to change it here

Reply:
This solution does not need the number of events.

Reply:
I make it an np.array after the loop okay? That way we still do not need the number of array, but can still use all the features of numpy

Reply:
Ok for me!